.github/workflows/release.yml

```diff
@@ -1,21 +1,21 @@
 name: Release
-
 on:
   push:
     tags:
       - '*'
-
+    branches: # TODO: Remove before merging.
+      - musl  # Pushing the feature branch should test this PR.
```

Comment:
Here is a run: https://github.com/dfinity/candid/actions/runs/5070940415
Here is the corresponding release.  It is marked as pre-release and can be deleted: https://github.com/dfinity/candid/releases/tag/musl